CMakeLists.txt

```diff
@@ -83,6 +89,41 @@ else()
     message(FATAL_ERROR "Unknown OS type")
 endif()
 
+if(NOT UMF_LINK_HWLOC_STATICALLY)
+    pkg_check_modules(LIBHWLOC hwloc>=2.3.0)
+    if(NOT LIBHWLOC_FOUND)
+        find_package(LIBHWLOC 2.3.0 REQUIRED hwloc)
+    endif()
+    # add PATH to DLL on Windows
+    set(DLL_PATH_LIST
+        "${DLL_PATH_LIST};PATH=path_list_append:${LIBHWLOC_LIBRARY_DIRS}/../bin"
+    )
+else()
+    include(FetchContent)
+    FetchContent_Declare(
+        hwloc_targ
+        GIT_REPOSITORY "https://github.com/open-mpi/hwloc.git"
+        GIT_TAG master
+        FIND_PACKAGE_ARGS)
+
+    FetchContent_GetProperties(hwloc_targ)
+    if(NOT hwloc_targ_POPULATED)
+        FetchContent_MakeAvailable(hwloc_targ)
+    endif()
+
+    set(hwloc_cmake ${hwloc_targ_SOURCE_DIR}/contrib/windows-cmake)
+
+    add_subdirectory(${hwloc_cmake} ${hwloc_targ_SOURCE_DIR})
+
+    set(LIBHWLOC_INCLUDE_DIRS ${hwloc_targ_SOURCE_DIR}/include)
+    set(LIBHWLOC_LIBRARY_DIRS ${hwloc_targ_SOURCE_DIR}/Debug)
```

Comment:
Does `Debug` means that hwloc is built in a debug mode?

Reply:
Corrected to Release